doc/example/store.ipynb

```diff
@@ -246,136 +436,115 @@
      },
      "metadata": {
       "needs_background": "light"
      },
      "output_type": "display_data"
     }
-   ],
-   "source": [
-    "# plot waterfalls\n",
-    "visualize.waterfall(result, size=(15,6))"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
     "### Save optimization result as HDF5 file\n",
     "\n",
-    "The optimization result can be saved via a `pypesto.store.OptimizationResultHDF5Writer`."
+    "The optimization result can be saved via a `pypesto.store.write_result()`. This will write the problem, the\n",
+    "optimization-,profile- and sampling_result to hdf5. All of them can be disabled through boolean values."
```

Comment:
```suggestion
    "The optimization result can be saved with `pypesto.store.write_result()`. This will write the problem and the\n",
    "optimization result, and the profiling and sampling results if available, to HDF5. All of them can be disabled with boolean flags."
```

Can add a link to the documentation for the function.